crypto/evp/pmeth_lib.c

```diff
@@ -29,6 +29,7 @@ static const EVP_PKEY_METHOD *standard_methods[] = {
 #ifndef OPENSSL_NO_DH
     &dh_pkey_meth,
 #endif
+    &pbkdf2_pkey_meth,
```

Comment:
I'll try to give it a shot and try to introduce "no-pbkdf2", if that's okay for you?